liquibase-core/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -100,8 +100,18 @@ private void populateValidateNullableIfNeeded(Column column, List<CachedRow> met
                 if (validated== null) {
                     break;
                 }
-                column.setShouldValidateNullable(validated.toString().equalsIgnoreCase(VALIDATE));
-                return;
```

Comment:
@neupsh  Did you find out why [RowMapperNotNullConstraintsResultSetExtractor ](https://github.com/Datical/liquibase/blob/ddb/liquibase-core/src/main/java/liquibase/executor/jvm/RowMapperNotNullConstraintsResultSetExtractor.java)didn't catch it ? 
I'm just curious where is my mistake 
I  thought that I implemented it for that puprose :-) and this logic is kind of duplication or seem like sort of that